### PR TITLE
RH Summit 2019 lab

### PR DESCRIPTION
**Pull request by ppatierno**
@mbogoevici this is a draft of the outline I had in mind using the material we already have.

### CODE REVIEW

**Comment by ppatierno** [comment]
@mbogoevici is it fine, do you want to propose changes or it's good to merge?

File: labs/mirror-maker-single-namespace.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Check on the OpenShift Web console that the related pods are not running anymore.
+
+=== Setting up the source and target clusters
+
+We will use the cluster previously created in this workshop as source but without the authentication enabled.
+In order to do that, let's update the already running `production-ready` cluster actually removing the authentication.
+
+----
+oc apply -f https://raw.githubusercontent.com/mbogoevici/workshop-amq-streams/master/configurations/clusters/production-ready.yaml
```
Comment (mbogoevici) [inline labs/mirror-maker-single-namespace.adoc]:
```suggestion
oc apply -f https://raw.githubusercontent.com/RedHatWorkshops/workshop-amq-streams/master/configurations/clusters/production-ready.yaml
```

File: labs/mirror-maker-single-namespace.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+----
+
+We will use this cluster as source of the mirroring, having the `timer-producer` application sending messages to the `lines` topic.
+
+Let's deploy another `production-ready-target` cluster as target of the mirroring from where the `log-consumer` application will read messages.
+
+----
+oc apply -f https://raw.githubusercontent.com/mbogoevici/workshop-amq-streams/master/configurations/clusters/production-ready-target.yaml
```
Comment (mbogoevici) [inline labs/mirror-maker-single-namespace.adoc]:
```suggestion
oc apply -f https://raw.githubusercontent.com/RedHatWorkshops/workshop-amq-streams/master/configurations/clusters/production-ready-target.yaml
```

File: labs/mirror-maker-single-namespace.adoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+----
+
+Wait for the new cluster to be deployed.
+
+Now, because the `timer-producer` application will write on the already existing `lines` topic on `production-ready` cluster, let's create a corresponding `lines` topic on the `production-ready-target` cluster as destination of the mirroring, from where the `log-consumer` application will read messages.
+
+----
+oc apply -f https://raw.githubusercontent.com/mbogoevici/workshop-amq-streams/master/configurations/topics/lines-10-target.yaml
```
Comment (mbogoevici) [inline labs/mirror-maker-single-namespace.adoc]:
```suggestion
oc apply -f https://raw.githubusercontent.com/RedHatWorkshops/workshop-amq-streams/master/configurations/topics/lines-10-target.yaml
```

**Comment by mbogoevici** [comment]
Rebased, merged into master

**Pull request closed without merging**